janitor/functions.py

```diff
@@ -55,6 +57,78 @@
 )
 
 
+def get_symbol(symbol: str) -> str:
+    """
+    This is a helper function to get a companies full
+    name based on the stock symbol.
+
+    Example:
+        print(get_symbol("aapl"))
+        console >> Apple Inc.
+
+    :param symbol: This is our stock symbol that we use
+        to query te api for the companies full name.
+    :return: This is the company name
+    """
+    result = requests.get(
+        "http://d.yimg.com/autoc."
+        + "finance.yahoo.com/autoc?query={}&region=1&lang=en".format(symbol)
+    ).json()
+
+    for x in result["ResultSet"]["Result"]:
+        if x["symbol"] == symbol:
+            return x["name"]
+        else:
+            return "Not found."
+
+
+def is_connected(url: str) -> bool:
```

Comment:
This is quite a nice function! Thanks for adding it in.

That said, I think it might be better placed in `utils.py`. What are your thoughts here?

Reply:
The more I think about it, the more I am convinced this is a utility function rather than a main data cleaning function.

**Action item:** Please move the function to `utils.py`.